src/cc/bcc_proc.c

```diff
@@ -149,21 +154,23 @@ int bcc_procutils_each_ksym(bcc_procutils_ksymcb callback, void *payload) {
   if (!kallsyms)
     return -1;
 
-  if (!fgets(line, sizeof(line), kallsyms)) {
```

Comment:
This line is gone......

Reply:
What the first line shows us? There must be a reason to ignore it in the first place?

Reply:
I don't know. It looks like totally normal symbol on my machine.....

Reply:
It looks like it depends on CONFIG_KALLSYMS_ALL.
If the config is on, all init, text and data symbols will be included.
Otherwise, data symbols are executed (except a few special symbols in the linker script file).